ds-caselaw-ingester/lambda_function.py

```diff
@@ -354,7 +378,7 @@ def handler(event, context):
     docx_filename = extract_docx_filename(metadata, consignment_reference)
     copy_file(
         tar,
-        f"{consignment_reference}/{docx_filename}",
+        f"{docx_filename}",
```

Comment:
This feels like it'd be a breaking change -- I can't find where we're changing the existing behaviour to embed the consignment_reference in the docx or move the file so it's not in a consignment_reference folder.

Reply:
Amended!